### ISSUE TITLE
App push notifications are broken

### ISSUE THREAD

**Issue by tiltec** [yunity/karrot-frontend#1102]
I noticed it a while ago that I didn't get app push notifications anymore. Janina didn't have the problem with a very old app version.

I assume it's caused by some frontend change, either the token is not sent to the backend or something about the cordova FCM plugin is broken.

**Comment by tiltec** [comment]
The last android push subscription was registered on 2018-06-18, so there's most likely something broken on the frontend. Need to investigate further.

While I did backend investigations, I noticed that we only react to the `InvalidRegistration` error, not the `NotRegistered` one. The latter is the one that happens when somebody uninstalls their app: https://firebase.google.com/docs/cloud-messaging/http-server-ref#error-codes
We should raise errors when any other error happens during push...
